.env.proxy.example

```
@@ -0,0 +1,26 @@
+BASE_DOMAIN=example.com
+
+# Optional: contact email for ACME certificate registrations
+ACME_EMAIL=admin@example.com
+
+# (Optional) Cloudflare token for DNS-01 challenge (if using Traefik or Caddy with DNS)
+# Requires permissions: Zone:DNS:Edit for the target zone
+CLOUDFLARE_API_TOKEN=cf_token_here
+
+# Explicit hostnames (can be omitted; will default to <service>.<BASE_DOMAIN>)
+PORTAL_HOSTNAME=portal.example.com
+N8N_HOSTNAME=n8n.example.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The N8N_HOSTNAME key should go before the PORTAL_HOSTNAME key

(UnorderedKey)